src/single_random_draw.rs

```diff
@@ -56,7 +56,7 @@ pub fn select_coins_srd<'a, R: rand::Rng + ?Sized, Utxo: WeightedUtxo>(
 
     result.clear();
 
-    let threshold = target + CHANGE_LOWER;
+    let threshold = target.checked_add(CHANGE_LOWER)?;
```

Comment:
With the next release of rust-bitcoin, Amount is limited to 21 million sats, so adding `CHANGE_LOWER` to Amount::MAX can't panic, so this commit can be dropped.